definitions/divorce/json/CaseField/CaseField.json

```diff
@@ -2571,15 +2571,15 @@
     "ID": "labelOtherProceedingsPetitioner",
     "Label": "### Petitioner's other proceedings:",
     "FieldType": "Label",
-    "SecurityClassification": "PUBLIC"
+    "SecurityClassification": "Public"
```

Comment:
just to make it consistent. "PUBLIC" and "Public" are the same for CCD. This field is not used at all...